apps/aecore/test/aecore_sync_SUITE.erl

```diff
@@ -1245,9 +1244,23 @@ encode_peer_for_config(Peer) ->
     Port = integer_to_binary(aec_peer:port(Peer)),
     <<"aenode://", PK/binary, "@", Host/binary, ":", Port/binary>>.
 
-random_peer() ->
+%% Make sure to generate non-overlapping ports...
+random_peers(N) ->
+    Ports = random_ports(N, []),
+    [ random_peer(Port) || Port <- Ports ].
+
+random_ports(0, Ports) -> Ports;
+random_ports(N, Ports) ->
+  Port = 4000 + rand:uniform(999),
```

Comment:
I'd go even further up to ~60000 for a pinch of more randomness

Reply:
Probably not neecessary, I don't even think the randomness brings _anything_ to the table. We could just as well have used 4001-4004 sequentially I think.